docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,306 @@
+of links in the comments for the pull request.
+You can then create an additional pull request for your each backport version you specified as a label.
 
 <Notice>
 
-The label method will fail for forks of `gravitational/teleport`.
+You must be an internal contributor to use the GitHub label process to backport changes.
+This method fails for forks of the `gravitational/teleport` repository.
 
 </Notice>
 
 </TabItem>
-<TabItem label="Using make">
+<TabItem label="Use make">
+
+To use make:
+
+1. Navigate to the root of your `gravitational/teleport` clone. 
 
-Navigate to the root of your `gravitational/teleport` clone and run the
-following command, assigning the `PR` variable to the ID of your GitHub PR and
+1. Run the following command to assign the `PR` variable to the ID of your original GitHub pull request and
 the `TO` variable to a comma-separated list of versioned branch names:
 
-```code
-$ PR=0000 TO=branch/v10 make backport
-```
+    ```code
+    $ PR=0000 TO=branch/v10 make backport
+    ```
 
-This script will automatically generate backport branches and open PRs for them.
+    This `make backport` script automatically generates backport branches and opens pull requests for them.
 
 <Notice>
 
-The `make backport` method will fail for forks of `gravitational/teleport`.
+You must be an internal contributor to use the GitHub label process to backport changes.
+This method fails for forks of the `gravitational/teleport` repository.
 
 </Notice>
 
 </TabItem>
-<TabItem label="Manually">
+<TabItem label="Select commits manually">
 
-If automatic backport methods have failed, you will need to backport manually.
+If automatic backport methods fail or you're an external contributor, you must backport changes manually.
 
-We will illustrate a backport for Teleport version 8.x.x. In this example, we
-have created a PR against `origin/master` from the branch `my-branch`, which has
-been committed to our fork.
+The following example illustrates how to backport a change into Teleport version 11.x.x. In this example:
 
-Copy the hash of the commit we made to `my-branch`, then run the following
-commands:
+- Changes were made in the `my-branch` branch and committed to a fork of the Teleport repository.
+- A pull request was created to merge the change from my-branch into `origin/master` from the fork.
 
-```code
-$ git fetch origin
-$ git switch branch/v8
-$ git switch -c my-branch-backport-v8
-$ git cherry-pick <commit-hash>
-$ git push -u fork my-branch-backport-v8
-```
+1. Copy the hash of the commit made to `my-branch`.
 
-Finally, create a PR for your backport.
+1. Run the following commands to create a branch for the commit being backported:
+
+    ```code
+    $ git fetch origin
+    $ git switch branch/v11
+    $ git switch -c my-branch-backport-v11
+    $ git cherry-pick <commit-hash>
+    ```
+
+1. Push the changes for the backport to the fork.
+
+    ```code 
+    $ git push -u fork my-branch-backport-11
```

Comment:
Above we used `--set-upstream`, but here we use `-u`. They're equivalent, but let's be consistent.

(I generally prefer the long form for written docs, as they're more self-documenting)

Reply:
👍